### ISSUE TITLE
ASP.NET Runtime missing from ARM32 tar.gz download on microsoft.com

### ISSUE THREAD

**Issue by couven92** [dotnet/cli#9343]
seealso #9218 

If you do not install the dotnet SDK via `dotnet-install` (which I never thought of doing since that never did work in the past for ARM), you'd go to the download page for [.NET Core](https://www.microsoft.com/net/download/dotnet-core/sdk-2.1.300-rc1) and select [`Linux`→`ARM32 Binaries`](https://www.microsoft.com/net/download/thank-you/dotnet-sdk-2.1.300-rc1-linux-arm32-binaries). This gives you a link to [`dotnet-sdk-2.1.300-rc1-008673-linux-arm.tar.gz`](https://download.microsoft.com/download/B/1/9/B19A2F87-F00F-420C-B4B9-A0BA4403F754/dotnet-sdk-2.1.300-rc1-008673-linux-arm.tar.gz).

Let's list the contents and grep for `AspNetCore`  

```
$ tar -tzf dotnet-sdk-2.1.300-rc1-008673-linux-arm.tar.gz | grep "AspNetCore"
./sdk/2.1.300-rc1-008673/Microsoft.AspNetCore.DeveloperCertificates.XPlat.dll
```

As can be seen from the output above, the ASP.NET Core runtime is missing. This causes the following error when attempting to run an ASP.NET Core application on ARM

```
It was not possible to find any compatible framework version
The specified framework 'Microsoft.AspNetCore.App', version '2.1.0-rc1-final' was not found.
  - Check application dependencies and target a framework version installed at:
      /opt/dotnet/
  - Installing .NET Core prerequisites might help resolve this problem:
      http://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409
  - The .NET Core framework and SDK can be installed from:
      https://aka.ms/dotnet-download
```

**Comment by dasMulli** [comment]
So in the links at https://github.com/dotnet/versions/tree/master/build-info/dotnet/product/cli/release/2.1-rtm it seems to be fixed for linux-arm but not linux-arm64

@natemcmaster do you know why there is no asp.net core runtime in the arm64 SDK but one in the arm (32bit) SDK?

**Comment by couven92** [comment]
@dasMulli huh? I am talking about the ARM32 binaries here! They do not have ASP.NET Runtime in them

**Comment by couven92** [comment]
@dasMulli Ah, I see, the download link https://github.com/dotnet/versions/tree/master/build-info/dotnet/product/cli/release/2.1-rtm is not the same as the one on https://www.microsoft.com/net/download/dotnet-core/sdk-2.1.300-rc1, but yes the link on GitHub links to an archive that include the ASP.NET Core runtime

**Comment by dasMulli** [comment]
yes early access for 2.1.0 RTM - see https://github.com/aspnet/Home/wiki/2.1.0-Early-Access-Downloads

the more official link would be https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.300/dotnet-sdk-2.1.300-linux-arm.tar.gz
however I just discovered that the arm64 one is missing the asp.net core runtimes https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.300/dotnet-sdk-2.1.300-linux-arm64.tar.gz

**Comment by couven92** [comment]
Ok, I do think, however, that fixing the links on the official Microsoft Download Page is also very important. IMO, the links should all consistently link to the same location

**Comment by natemcmaster** [comment]
This looks like a duplicate of https://github.com/dotnet/cli/issues/9162. Arm32 was fixed after rc1 with https://github.com/dotnet/cli/pull/9163. 

Aspnet does not yet support arm64. See https://github.com/aspnet/Universe/issues/1117

**Comment by natemcmaster** [comment]
@couven92 we haven't released 2.1.0 RTM yet. Those links will update when we do :)
